burn-import/src/onnx/from_onnx.rs

```diff
@@ -534,6 +558,58 @@ fn lift_constants(nodes: &mut Vec<Node>) {
     );
 }
 
+/// Handle Identity nodes.
+///
+/// There are two types of Identity nodes:
+/// 1. Pass-through nodes that are used to connect two nodes. These are removed from the graph.
+/// 2. Nodes that act as a constant (its input has initializer). Change the node type to Constant.
+fn handle_identity(nodes: &mut Vec<Node>) {
+    log::info!("Handling identity nodes");
+
+    let mut identity_nodes_to_remove = Vec::new();
+
+    for node in nodes
+        .iter_mut()
+        .filter(|node| node.node_type == NodeType::Identity)
+    {
+        // if the node has states, it is a constant. Move the data to value attribute for consistency.
+        if let Some(state) = node.states.first() {
+            match &state.ty {
+                // Currently there is only tensor type
+                StateType::Tensor(tensor) => {
+                    node.attrs
+                        .insert("value".to_string(), AttributeValue::Tensor(tensor.clone()));
+                    node.states.clear();
+                }
+            }
+            node.node_type = NodeType::Constant;
+            log::debug!("Converted identity node ({}) to constant", node.name);
+        } else {
+            // Support pass through identity node
+            identity_nodes_to_remove.push(node.clone());
+        }
+    }
+
+    // Remove the identity nodes that are only used to connect two nodes
+    for identity_node in identity_nodes_to_remove.iter() {
+        let input = identity_node.inputs.first().unwrap();
+        let output = identity_node.outputs.first().unwrap();
```

Comment:
I think we should replace the unwrap with an expect here: `expect("Identity node should have one input.")` and `expect("Identity node should have one output.")`